lldb/unittests/Expression/ValueMatcher.cpp

```diff
@@ -0,0 +1,202 @@
+//===-- ValueMatcher.cpp --------------------------------------------------===//
```

Comment:
```suggestion
//===----------------------------------------------------------------------===//
```
(Since https://discourse.llvm.org/t/is-c-in-header-files-still-relevant/83124)